tests/integration/idmap.bats

```diff
@@ -2,167 +2,329 @@
 
 load helpers
 
+OVERFLOW_UID="$(cat /proc/sys/kernel/overflowuid)"
+OVERFLOW_GID="$(cat /proc/sys/kernel/overflowgid)"
+
```

Comment:
I am not sure it is well defined in bats when exactly this code is executed, so this kind of sticks out.

So maybe move it to `function setup()` for clarity (or, if you really want to be only executed once, to `helpers.bash`, which is, unlike this file, a bash script).

Reply:
Fair enough.